src/controllers/QueryRunner.ts

```diff
@@ -339,6 +344,22 @@ export default class QueryRunner {
         return !!includeHeaders;
     }
 
+    private shouldRemoveNewLines(): boolean {
+        // get config copyRemoveNewLine option from vscode config
+        let config = this._vscodeWrapper.getConfiguration(Constants.extensionConfigSectionName);
+        let removeNewLines: boolean = config[Constants.configCopyRemoveNewLine];
+        return removeNewLines;
+    }
+
+    private removeNewLines(inputString: string): string {
+        // This regex removes all newlines in all OS types
+        // Windows(CRLF): \r\n
+        // Linux(LF): \n
+        // MacOS(CR): \r
```

Comment:
This comment isn't exactly right, old mac os using /r (pre osx) but osx now follows the same new lines as linux https://en.wikipedia.org/wiki/Newline